crates/moonrun/src/sys_api.rs

```diff
@@ -106,6 +110,7 @@ fn get_env_vars(
 pub fn init_env<'s>(
     obj: v8::Local<'s, v8::Object>,
     scope: &mut v8::HandleScope<'s>,
+    program: &str,
```

Comment:
I thougth `program` is `moonrun`(instead of `xxx.wasm`) at first glance. maybe a more suitable arg name?

Reply:
`executable`?